src/api.ts

```diff
@@ -0,0 +1,9 @@
+// eslint-disable-next-line max-len
+const BASE_URL = 'https://mate-academy.github.io/react_people-table/api/people.json';
+
+function getData<T>(): Promise<T> {
+  return fetch(BASE_URL)
+    .then(response => response.json());
+}
+
+export const getPeople = () => getData<Person[]>();
```

Comment:
you also need to implement this part of the task:
"Find a mother and a father by motherName and fatherName and add them to the person for future use"

Reply:
Why should I search for mother and father if I have them after getting them from the API?

Reply:
it seems we will need it for the next task
https://share.getcloudapp.com/xQuw88Ok